lib/stages/coding_style.sh

```diff
@@ -18,6 +18,7 @@ _stage_coding_style() {
 __test_php_coding_styles() {
     printf "Checking php coding styles\n\n"
 
+    phpcs --config-set installed_paths ../../drupal/coder/coder_sniffer
     phpcs -ps --standard=Drupal --extensions=php,module,inc,install,test,profile,theme --ignore="${DRUPAL_TESTING_PHPCS_IGNORE_PATTERN}" .
```

Comment:
Instead of making assumptions about paths we could make this more robust by doing...

`--runtime-set installed_paths $(composer config -g --absolute vendor-dir)/drupal/coder/coder_sniffer`

on the phpcs commands. Then we don't alter any global state and things can move. Maybe want to drop the -g and then we use the coder version installed by the package we're testing.

Reply:
Ah but you want to be able to fail quickly. So yeah we have to assume global installation.

Reply:
This is a very nice idea! but for some reason the global vendor dir cannot be retrieved, see:
https://travis-ci.com/github/thunder/drupal-testing/jobs/340350024#L536
I could use the home directory though, but that would even be less robust than the relative version I tried first